### ISSUE TITLE
Looking for collaborators.

### ISSUE THREAD

**Issue by meskyanichi** [meskyanichi/mongoid-paperclip#63]
Since I haven't used Mongoid and/or Paperclip for quite a while, I'd like to add one or more collaborators that are still actively using this setup and would like to handle PR and issues. Comment below if you're interested.

**Comment by yordis** [comment]
@meskyanichi I am in some project using a lot but I would say I need more experience working with it.

**Comment by meskyanichi** [comment]
Any help is much appreciated in any case. It's been years since I've used MongoDB so I'm not up-to-date with potential issues that might arise today. If you're actively using this gem then I'll gladly make you a collaborator. Often times people submit pull requests with fixes in any case, so just negotiating/pulling in the pull requests would be a lot of help. Let me know.

**Comment by yordis** [comment]
@meskyanichi I can help you, I will accept PR if I know what's going on, again, I don't want to mess something because my ignorance but for sure I can help whenever I can do it

**Comment by meskyanichi** [comment]
@yordis Thanks! I've added your as a collaborator.

If anyone else is interested, comment below.

**Comment by tilsammans** [comment]
@mrrooijen yeah i'm interested! do you feel comfortable with adding me as a collaborator on rubygems too? It'd be nice to release a new gem version and get the ball rolling again.

**Comment by mrrooijen** [comment]
@tilsammans Thanks! I've added you as a collaborator to both this repository as well as rubygems so you can push commits and release gems. 👍

**Comment by tilsammans** [comment]
Thanks! I'll take good care of 'er! 😄
